control-plane-operator/controllers/hostedcontrolplane/cloud/aws/params.go

```diff
@@ -31,5 +35,21 @@ func NewAWSParams(hcp *hyperv1.HostedControlPlane) *AWSParams {
 	}
 	p.OwnerRef = config.ControllerOwnerRef(hcp)
 
+	p.DeploymentConfig.SetDefaults(hcp, ccmLabels(), pointer.Int(1))
```

Comment:
This is the main difference.  The rest is a refactor to make it more like the param/reconcile pattern in the other components.